components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClientConstants.java

```diff
@@ -155,7 +155,7 @@
     public static final String API_VERSION_CONTEXT_METHOD_FACET ="api_version_context_method_facet";
     public static final String API_VERSION_CONTEXT_DEST_FACET="api_version_context_dest_facet";
     public static final String API_VERSION_APIPUBLISHER_CONTEXT_FACET ="api_version_apiPublisher_context_facet";
-    public static final String API_APIPUBLISHER_APPLICATIONNAME_FACET="api_apiPublisher_applicationName_facet";
+    public static final String API_YEAR_MONTH_WEEK_DAY_FACET ="api_year_month_week_day_facet";
```

Comment:
fixed in commit: https://github.com/ruks/carbon-apimgt/commit/d7e56b1d85254369d07aab9572f13d4c2120df97